### ISSUE TITLE
Better schema-common inheritance

### ISSUE THREAD

**Issue by gurhar1133** [radiasoft/sirepo#4729]
Currently adding a field onto schema-common.model causes errors. 

If an individual app model wants to inherit from schema common, this is a problem since inheritance only allows inheriting from the same section (ie. a model on app-schema.model must inherit from app-schema.model).

Either adding to schema-common.model should not cause errors, or inheritance should be more flexible in some other way

**Comment by robnagler** [comment]
It's not clear to me that this is inheritance, and instead, I think it is aggregation or inclusion.

I want to address this with the new GUI. I think the approach needs to be quite different.

**Comment by gurhar1133** [comment]
> It's not clear to me that this is inheritance, and instead, I think it is aggregation or inclusion.
> 
> I want to address this with the new GUI. I think the approach needs to be quite different.

Yes, maybe not technically inheritance, just using that language because of "_super", and I think @moellep also thought this should be for new GUI.
